feat(cli): auto-detect current git branch for download and upload commands (#126)
transi-store/transi-store: packages/cli/README.md

#### Options

- `--config <path>`: Path to the configuration file (default: `transi-store.config.json`).

> - `--branch`: The branch to download the translations from. Defaults to "main". Use `@all` to download all translations across every branch (no branch filtering). If omitted and you are inside a git repository on a feature branch, the current branch is detected and used automatically.

Reviewer: ```suggestion
- `--branch`: The branch to download the translations from. Defaults to the current branch in a git environment. Use `@all` to download all translations across every branch (no branch filtering).
```
